src/mjolnir/ferry_connections.cc

```diff
@@ -42,7 +44,8 @@ uint32_t ShortestPath(const uint32_t start_node_idx,
                       sequence<Edge>& edges,
                       sequence<Node>& nodes,
                       const bool inbound,
-                      const uint32_t rc) {
+                      const uint32_t rc,
+                      uint32_t& mode_count) {
```

Comment:
this counts the total number of expansion we had to do over all nodes in the tileset